components/org.wso2.carbon.identity.oauth.stub/src/main/resources/OAuthAdminService.wsdl

```diff
@@ -1,14 +1,14 @@
 
```

Comment:
Just to know, How did we create the updated WSDL?

Reply:
First wrote the adminService back-end, then patched it by making false the hide admin service in WSO2 product. Then copy that wsdl to here. Am I wrong @mefarazath ?